bson/src/main-legacy/org/bson/BasicBSONEncoder.java

```diff
@@ -190,15 +189,6 @@ protected void _putObjectField(final String name, final Object initialValue) {
             putCodeWScope(name, (CodeWScope) value);
         } else if (value instanceof Code) {
             putCode(name, (Code) value);
-        } else if (value instanceof DBRef) {
```

Comment:
This can't be here any more since DBRef isn't accessible from bson.  The upside though is that the DefaultDBEncoder subclass handles DBRef in its putSpecial method.

Reply:
Ack